locales/fr/json.json

```diff
@@ -512,7 +512,7 @@
     "Mozambique": "Mozambique",
     "Multiple Months": "Plusieurs mois",
     "Myanmar": "Myanmar",
-    "N/A": "N/A",
+    "N/A": "N / A",
```

Comment:
@caouecs, Google Translate for French constantly suggests this change. Is this correct or not?

Reply:
ND (for "non disponible") or NC (for "non communiqué" / "non connu") would be a better translation.

Reply:
@caouecs, @WhereIsLucas, what do you think?

Reply:
@caouecs, can you help?

Reply:
I agree with @wblondel

Reply:
```suggestion
    "N/A": "n.c.",
```